src/components/PersonCards.js

```diff
@@ -41,8 +38,9 @@ export const PersonCard = ({ person, menu, subtitle, slug }) => {
     </A>
     <div className='person-body'>
       <A className='name' to={`/u/${id}`}>{name}</A>
-      <div className='role'>{role_and_age}</div>
-      {subtitle && <div className='subtitle'>{subtitle}</div>}
+      <div className='role'>
+        {(isModerator ? 'Moderator' : 'Member') + ` since ${humanDate(joined_at)}`}
```

Comment:
not understanding this. If I skip the braces it just prints the literal code in the page, doesn't evaluate it...